test/test_facebook.py

```diff
@@ -351,5 +356,78 @@ def test_get_user_permissions_nonexistant_user(self):
             facebook.GraphAPI(token).get_permissions(1)
 
 
+class TestAppSecretProof(FacebookTestCase):
+
+    proof = '4dad02ff1693df832f9c183fe400fc4f601360be06514acb4a73edb783eec345'  # noqa
+
+    def test_appsecret_proof_set(self):
+        api = facebook.GraphAPI(access_token='abc123', app_secret='xyz789')
```

Comment:
Instead of `api`, can we call it `graph`? This is more consistent with our other code and documentation.